pvlib/snow.py

```diff
@@ -185,3 +185,104 @@ def dc_loss_nrel(snow_coverage, num_strings):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     '''
     return np.ceil(snow_coverage * num_strings) / num_strings
+
+def townsend_Se(S, N):
+    '''
+    Calculates effective snow for a given month based upon the total snowfall
+    received in a month in inches and the number of events where snowfall is greater
+    than 1 inch
+
+    Parameters
+    ----------
+    S : numeric
+        Snowfall in inches received in a month
+
+    N: numeric
+        Number of snowfall events with snowfall > 1"
+
+    Returns
+    -------
+    effective_snowfall : numeric
+        Effective snowfall as defined in the townsend model 
+
+    References
+    ----------
+    .. [1] Townsend, Tim & Powers, Loren. (2011). Photovoltaics and snow: An 
+            update from two winters of measurements in the SIERRA. Conference 
+            Record of the IEEE Photovoltaic Specialists Conference. 
+            003231-003236. 10.1109/PVSC.2011.6186627. 
+       Available at https://www.researchgate.net/publication/261042016_Photovoltaics_and_snow_An_update_from_two_winters_of_measurements_in_the_SIERRA
+
+    '''
+    return(np.where(N>0, 0.5 * S * (1 + 1/N), 0))
+
+def townsend_snow_loss_model(S, N, tilt, RH, T_air, POA, R, H, P=40):
+    '''
+    Calculates monthly snow loss based on a generalized monthly snow loss model discussed in [1]_.
+
+    Parameters
+    ----------
+    S : numeric
+        Inches of snow received in the current month
+
+    N : numeric
+        Number of snowfall events with snowfall > 1"
+
+    tilt : numeric
+        Array tilt in degrees
+
+    RH : numeric
+        Relative humidity in percentage
+
+    T_air : numeric
+        Ambient temperature in celcius
```

Comment:
```suggestion
    temp_air : numeric
        Ambient temperature [C].
```

Where these variables are already in pvlib we should use the same parameter name.

Reply:
Do we have a document with existing variable names?

Reply:
> Do we have a document with existing variable names?

Yes, although it would probably benefit from an update: https://pvlib-python.readthedocs.io/en/latest/variables_style_rules.html